reactapp/src/components/UnorderedList/UnorderedList.js

```diff
@@ -0,0 +1,8 @@
+export const UnorderedList = ({arrayElements}) => {
+    return (
+      <ul>
+        {arrayElements.map((arrayElements, index) =>
```

Comment:
tutaj powinno byc:

 {arrayElements.map((arrayElement, index) =>
       <li key={`elem-${index}`}>{arrayElement}</li>
       
"arrayElements" to nazwa listy po ktorej iterujesz, natomiast nazwa pojedynczego elementu jaki przekazujesz do LI powinna byc juz inna